protocols/tls13/src/cryptolib.rs

```diff
@@ -0,0 +1,271 @@
+// A module that wraps all the Generic crypto needed by TLS 1.3
+// Each function below should be supported by a standard crypto library
+
+// Import hacspec and all needed definitions.
+use hacspec_lib::*;
+
+use hacspec_sha256::*;
+// XXX: this type of import is not allowed in hacspec
```

Comment:
Indeed. This is only for implementation reasons. If we allow that, then I have to carry around a map of import renamings for Hacspec typechecking. And then I would have to do alpha-renaming in the F* backend.

Reply:
The issue is that the crates currently implement functions like `sha256::hash`. So we get a very generic `hash` function. We can get rid of this by only exporting "fully qualified" function names in the other specs.